app/views/donors/index.html.slim

```diff
@@ -18,16 +18,16 @@
                 .col-xs-6.col-sm-6.col-md-4
                   = select_tag :cause_id, options_from_collection_for_select(Cause.all, :id, :name, params[:cause_id]), prompt: "Filter by cause:", class: 'form-control', onchange: "cause_toggle(this.value)"
         tr
-          th Name
-          th NRIC/UEN
+          th 
+            | Name
+            = sortable "name"
```

Comment:
You added the method `#sortable`, so you should change that method to accept a symbol. 😀 

This is done by converting the argument to a string inside the method using `#to_s`.